lib/components/CentralDevice.jsx

```diff
@@ -133,6 +134,7 @@ class CentralDevice extends React.PureComponent {
         const advIconTitle = advertising ? 'Advertising' : 'Not advertising';
         const iconCheckmarkConnUpdate = autoConnUpdate ? 'icon-ok' : '';
         const iconCheckmarkPairing = (security && security.autoAcceptPairing) ? 'icon-ok' : '';
+        const deviceType = deviceTypeDefinitions[deviceInfo.deviceType];
```

Comment:
Perhaps use `deviceTypeDefinitions[deviceInfo.deviceType] || 'Unknown device'` in case pc-nrfjprog-js returns a device type that is not in `deviceTypeDefinitions`.

Reply:
Yes, good suggestion!